stdlib/public/TensorFlow/Ops.swift

```diff
@@ -1700,14 +1723,18 @@ extension PartialRangeThrough : TensorSliceIndexProtocol where Bound == Int {
 }
 
 public extension Tensor {
-  public struct IndexPath {
-    public let begin, end, strides: Tensor<Int32>
-    public let beginMask, endMask, ellipsisMask, newAxisMask, squeezeAxisMask: Int64
-    
+  struct IndexPath {
```

Comment:
To make this fully GPE-compliant, the struct itself also needs to be `@_fixed_layout` and `@usableFromInline`.

Reply:
This struct is public (it's properties are not). In this case, should I just add `@_fixed_layout`?

Reply:
Nvm I made the struct itself internal too.